doc/nrf/ug_matter_configuring_env.rst

```diff
@@ -11,19 +11,19 @@ Setting up Matter development environment is all about setting up the Matter net
 In Matter, the commissioning procedure takes place over Bluetooth LE between a Matter accessory device and the Matter controller, where the controller has the commissioner role.
 When the procedure has completed, the device should be equipped with all information needed to securely operate in the Matter network.
 
-During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the Thread network credentials to the Matter accessory device.
-As a result, the device can join the Thread network and communicate with other Thread devices in the network.
+During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the IPv6 network credentials to the Matter accessory device.
```

Comment:
```suggestion
During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the Thread or WiFi network credentials to the Matter accessory device.
```
While both Thread and WiFi are technologies that IPv6 can use, the security component only exists at the underlying layer, so "IPv6 network credentials" term may be confusing.